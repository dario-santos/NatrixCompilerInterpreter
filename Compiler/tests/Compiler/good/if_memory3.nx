val x : int = 50;

if(x == 1)
{
    printn(x);
    val y : int = 201; 
    printn(x);

    printn(y);
}
else if(x == 2)
{
    val w : int = 544; 
    printn(x);

    printn(w);

}
else if(x == 3)
{
    val v : int = 789; 
    printn(x);

    printn(v);

}
else
{
    val p : int = 123132;

    printn(x);

    printn(p);
}

printn(x);

val z : int = 99;

printn(z);
