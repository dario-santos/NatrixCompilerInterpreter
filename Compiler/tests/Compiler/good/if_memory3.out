50
123132
50
99
